src/main.d.ts

```diff
@@ -0,0 +1,6 @@
+import { Auth } from '@firebase/auth';
+
+export interface IInitializeParams{
+  firebaseAuth?: Auth,
+  forceSignInAnonymously: boolean,
+}
```

Comment:
👍 
What do you think about add types also for Analytics object? Just an idea :)
```
export declare namespace Analytics {
  function initialize(params?: unknown): void
  function trackView(viewName: string, properties: Record<string, unknown>, modules?: string[]): void
  function trackEvent(eventName: string, properties: Record<string, unknown>, modules?: string[]): void
  function trackConversion(conversionLabel: string): void
  function setGlobalProperties(property: string, value: Record<string, unknown> | string): void
}
```

Reply:
In fact it's very good idea, thanks! I'll do it! :) 
But if it's an object, then why not an Interface instead of namespace?

Reply:
Yup, it could be namespace, but better idea is declare const. It can't be interface because Analytics object is already created :)

Reply:
I'm not sure if I understand you correctly, but I already created an `IAnalytics` interface in the main.d.ts and added `@type` to the exported Analytics const in the index.js ;) Is that what you meant? :D

Reply:
done in #44e4dee :)